## **Contextualização e Escopo do Projeto**

### **Contexto**

A **Kevin Cookie Company (KCC)** deseja melhorar suas vendas utilizando análises de dados baseadas no seu banco de dados SQL. Seu papel como consultor é:

1. Extrair e processar os dados para entender o desempenho atual da empresa.
2. Fornecer insights claros e responder às perguntas de negócio.
3. Apresentar um relatório interativo com gráficos e recomendações.

### **Objetivo do Projeto**

Responder às seguintes perguntas de negócio usando **Python**, **SQL**, e bibliotecas gráficas modernas:

1. **Clientes**:
    
    - Quem são os **3 melhores clientes** da empresa?
    - Por que eles são os melhores? Como fidelizá-los e aumentar suas compras?
2. **Produtos**:
    
    - Quais são os **melhores** e **piores produtos**?
    - Quais **novos sabores** podem substituir os piores produtos?
3. **Sazonalidade**:
    
    - Existe **sazonalidade** nas vendas?
    - Qual campanha de marketing pode ser criada para aproveitar essa sazonalidade?
4. **Preço**:
    

- Quais **produtos devem aumentar de preço**?
- Qual o impacto esperado nos ganhos com o novo preço?

## **Análise Prévia do Banco de Dados**

### **Tabelas**

- **Customers**: Informações dos clientes (nome, endereço, etc.).
- **Orders**: Detalhes dos pedidos realizados.
- **Order\_Product**: Relação entre pedidos e produtos (quantidade, valores).
- **Product**: Informações dos produtos (nome, preço, etc.).

### **Relacionamentos**

- A tabela **Orders** é relacionada com:
    - **Customers** (clientes que fizeram pedidos).
    - **Order\_Product** (itens de cada pedido).
    - **Product** (detalhes dos produtos vendidos).

### **Análise Exploratória**

In [4]:
-- listando todas as tabelas
SELECT TABLE_NAME 
FROM KCC.INFORMATION_SCHEMA.TABLES
WHERE TABLE_TYPE = 'BASE TABLE';


(5 rows affected)

Total execution time: 00:00:00.012

TABLE_NAME sysdiagrams Orders Product Order_Product Customers

In [5]:
-- Listando as colunas de todas as tabelas
SELECT TABLE_NAME, COLUMN_NAME, DATA_TYPE
FROM KCC.INFORMATION_SCHEMA.COLUMNS;


(25 rows affected)

Total execution time: 00:00:00.027

TABLE_NAME COLUMN_NAME DATA_TYPE sysdiagrams name nvarchar sysdiagrams principal_id int sysdiagrams diagram_id int sysdiagrams version int sysdiagrams definition varbinary Orders OrderID int Orders OrderDate date Orders CustomerID int Orders OrderTotal money Product CookieID int Product CookieName nvarchar Product RevenuePerCookie money Product CostPerCookie money Order_Product OrderID int Order_Product CookieID int Order_Product Quantity int Customers CustomerID int Customers CustomerName nvarchar Customers Phone nvarchar Customers Address nvarchar Customers City nvarchar Customers State nvarchar Customers Zip nvarchar Customers Country nvarchar Customers Notes nvarchar

In [6]:
--Contando o número de registros em cada tabela
SELECT 'Customers' AS TableName, COUNT(*) AS TotalRecords FROM KCC.dbo.Customers
UNION ALL
SELECT 'Orders', COUNT(*) FROM KCC.dbo.Orders
UNION ALL
SELECT 'Order_Product', COUNT(*) FROM KCC.dbo.Order_Product
UNION ALL
SELECT 'Product', COUNT(*) FROM KCC.dbo.Product;


(4 rows affected)

Total execution time: 00:00:00.015

TableName TotalRecords Customers 6 Orders 50 Order_Product 131 Product 6

In [7]:
--Analisando a tabela Custumers
SELECT * FROM KCC.dbo.Customers;


(6 rows affected)

Total execution time: 00:00:00.003

CustomerID CustomerName Phone Address City State Zip Country Notes 1 Tres Delicious 999-999-9999 123 Main Street Seattle WA 98112 United States High maintenance, not sure if it's worth having them as a customer, but don't tell them that! 2 ABC Groceries 801-583-8695 3215 Tori Lane Salt Lake City UT 84113 United States Friendly but a little old school. Talks way too much, set a time limit on calls. 3 ACME Bites 920-419-6270 4660 Sycamore Lake Road Green Bay WI 54303 United States One of our best customers! Likes to talk about roses. Make sure to read all about roses on Wikipedia before meeting. 4 Wholesome Foods 347-789-7688 1521 Redbud Drive Huntington NY 11743 United States CEO has three kids, first one is Charles, don't know the rest. CEO cares about Charles the most, so all good. 5 Park & Shop Convenience Stores 251-655-2909 2217 Lonely Oak Drive Mobile AL 36602 United States New customer, make sure to please them! Very frugal, so reinforce how delicious AND inexpensive our cookies are. 6 Tres Delicious 999-999-9999 12 Cité Dupetit-Thouars Paris 75003 France East Coast HQ.

In [8]:
--Analisando as cidades, estados e países distintos da tabela Customers
SELECT 
    DISTINCT City, 
    State, 
    Country
FROM 
    KCC.dbo.Customers
ORDER BY 
    Country, State, City;


(6 rows affected)

Total execution time: 00:00:00.003

City State Country Paris France Mobile AL United States Huntington NY United States Salt Lake City UT United States Seattle WA United States Green Bay WI United States

In [9]:
--Analisando a tabela Orders
SELECT * FROM KCC.dbo.Orders;;

(50 rows affected)

Total execution time: 00:00:00.012

OrderID OrderDate CustomerID OrderTotal 1 2022-01-01 5 1815,00 2 2022-01-02 5 964,00 3 2022-01-04 2 2238,00 4 2022-01-06 1 737,00 5 2022-01-09 4 569,00 6 2022-01-09 4 1586,00 7 2022-01-09 4 210,00 8 2022-01-11 5 2424,00 9 2022-01-13 2 1982,00 10 2022-01-13 4 39,00 11 2022-01-15 5 1383,00 12 2022-01-19 1 1680,00 13 2022-01-20 3 3293,00 14 2022-01-21 1 773,00 15 2022-01-21 3 1093,00 16 2022-01-21 2 998,00 17 2022-01-23 3 912,00 18 2022-01-24 4 915,00 19 2022-01-27 2 1410,00 20 2022-02-01 2 1955,00 21 2022-02-05 5 74,00 22 2022-02-05 4 1744,00 23 2022-02-06 2 1776,00 24 2022-02-06 3 1829,00 25 2022-02-07 4 1113,00 26 2022-02-08 3 171,00 27 2022-02-09 3 1291,00 28 2022-02-11 4 2060,00 29 2022-02-11 5 1263,00 30 2022-02-12 1 2117,00 31 2022-02-13 2 1900,00 32 2022-02-16 5 2847,00 33 2022-02-17 3 2159,00 34 2022-02-18 2 1196,00 35 2022-02-21 5 1165,00 36 2022-02-25 4 1391,00 37 2022-02-25 2 1415,00 38 2022-02-26 1 685,00 39 2022-02-26 5 1094,00 40 2022-02-28 1 487,00 41 2022-02-28 5 240,00 42 2022-03-07 1 1351,00 43 2022-03-07 5 3518,00 44 2022-03-07 3 952,00 45 2022-03-08 3 2208,00 46 2022-03-09 5 648,00 47 2022-03-10 3 798,00 48 2022-03-12 4 655,00 49 2022-03-13 1 588,00 50 2022-03-14 4 180,00

In [10]:
--Distribuição de pedidos por mês
SELECT 
    MONTH(OrderDate) AS Month,                  
    COUNT(*) AS TotalOrders,                    
    SUM(OrderTotal) AS TotalRevenue             
FROM 
    KCC.dbo.Orders
GROUP BY 
    MONTH(OrderDate)                           
ORDER BY 
    Month;                                     


(3 rows affected)

Total execution time: 00:00:00.004

Month TotalOrders TotalRevenue 1 19 25021,00 2 22 29972,00 3 9 10898,00

In [11]:
--Analisando a tabela Order_Product
SELECT * FROM KCC.dbo.Order_Product;

(131 rows affected)

Total execution time: 00:00:00.002

OrderID CookieID Quantity 1 1 229 1 2 160 1 3 84 1 5 30 2 4 49 2 6 128 3 1 82 3 2 170 3 3 202 3 5 180 3 6 18 4 2 209 4 3 25 4 4 82 4 5 25 5 1 46 5 5 11 5 6 51 6 1 214 6 3 24 6 6 66 7 2 210 8 2 114 8 3 36 8 4 186 8 5 106 8 6 178 9 1 119 9 2 226 9 3 153 9 4 99 10 2 39 11 2 53 11 4 235 11 6 65 12 1 171 12 2 234 12 5 197 13 1 201 13 2 193 13 3 239 13 4 225 14 2 227 14 3 78 14 6 26 15 1 61 15 2 99 15 3 109 15 5 48 16 4 56 16 6 129 17 3 63 17 5 199 18 3 183 19 6 235 20 1 244 20 3 147 21 2 74 22 2 141 22 3 176 22 5 241 23 1 132 23 4 27 23 6 168 24 2 179 24 5 122 24 6 214 25 2 239 25 3 42 25 4 166 26 2 171 27 1 179 27 6 66 28 4 134 28 5 172 28 6 168 29 2 79 29 3 96 29 4 80 29 5 94 29 6 17 30 1 169 30 4 204 30 5 152 31 1 151 31 3 229 32 1 93 32 3 128 32 4 122 32 6 209 33 3 103 33 5 202 33 6 173 34 1 200 34 2 196 35 3 34 35 4 164 35 5 113 36 1 116 36 4 127 36 5 101 37 1 136 37 5 245 38 1 137 39 4 14 39 6 173 40 3 87 40 4 13 41 1 48 42 2 123 42 4 217 42 5 120 43 1 209 43 3 125 43 4 18 43 5 220 43 6 186 44 2 155 44 4 20 44 5 239 45 4 168 45 5 224 45 6 144 46 2 177 46 4 66 46 5 69 47 4 132 47 6 45 48 3 131 49 4 147 50 6 30

In [12]:
--Analisando a tabela Product
SELECT * FROM KCC.dbo.Product;

(6 rows affected)

Total execution time: 00:00:00.001

CookieID CookieName RevenuePerCookie CostPerCookie 1 Chocolate Chip 5,00 2,00 2 Fortune Cookie 1,00 0,50 3 Oatmeal Raisin 5,00 2,20 4 Snickerdoodle 4,00 1,50 5 Sugar 3,00 1,25 6 White Chocolate Macadamia Nut 6,00 2,75

In [13]:
--Estatísticas descritivas para os valores numéricos
--Tabela Orders - Variável: OrderTotal
SELECT 
    MIN(OrderTotal) AS MinOrderTotal,
    MAX(OrderTotal) AS MaxOrderTotal,
    AVG(OrderTotal) AS AvgOrderTotal,
    SUM(OrderTotal) AS TotalOrderValue,
    STDEV(OrderTotal) AS StdDevOrderTotal
FROM KCC.dbo.Orders;



(1 row affected)

Total execution time: 00:00:00.005

MinOrderTotal MaxOrderTotal AvgOrderTotal TotalOrderValue StdDevOrderTotal 39,00 3518,00 1317,82 65891,00 797,812869727006

In [14]:
--Estatísticas descritivas para os valores numéricos
--Tabela Order_Product - Variável: Quantity
SELECT 
    MIN(Quantity) AS MinQuantity,
    MAX(Quantity) AS MaxQuantity,
    AVG(Quantity) AS AvgQuantity,
    SUM(Quantity) AS TotalQuantitySold,
    STDEV(Quantity) AS StdDevQuantity
FROM KCC.dbo.Order_Product;


(1 row affected)

Total execution time: 00:00:00.003

MinQuantity MaxQuantity AvgQuantity TotalQuantitySold StdDevQuantity 11 245 131 17249 68,90678684315185

In [15]:
--Estatísticas descritivas para os valores numéricos
--Tabela Product - Variáveis: RevenuePerCookie
SELECT 
    MIN(RevenuePerCookie) AS MinRevenue,
    MAX(RevenuePerCookie) AS MaxRevenue,
    AVG(RevenuePerCookie) AS AvgRevenue,
    SUM(RevenuePerCookie) AS TotalRevenue,
    STDEV(RevenuePerCookie) AS StdDevRevenue
FROM KCC.dbo.Product;


--Tabela Product - Variáveis: CostPerCookie
SELECT 
    MIN(CostPerCookie) AS MinCost,
    MAX(CostPerCookie) AS MaxCost,
    AVG(CostPerCookie) AS AvgCost,
    SUM(CostPerCookie) AS TotalCost,
    STDEV(CostPerCookie) AS StdDevCost
FROM KCC.dbo.Product;



(1 row affected)

(1 row affected)

Total execution time: 00:00:00.014

MinRevenue MaxRevenue AvgRevenue TotalRevenue StdDevRevenue 1,00 6,00 4,00 24,00 1,7888543819998317

MinCost MaxCost AvgCost TotalCost StdDevCost 0,50 2,75 1,70 10,20 0,7905694150420949

## **Respondendo as Perguntas de Negócios**

### **Pergunta 1: Quem são os 3 melhores clientes da empresa e por quê?**

In [20]:
-- Selecionando os 3 melhores clientes de acordo com o total de gastos
SELECT TOP 3 
    C.CustomerID, 
    C.CustomerName, 
    SUM(O.OrderTotal) AS TotalSpent,
    COUNT(O.OrderID) AS TotalOrders
FROM KCC.dbo.Customers C
INNER JOIN KCC.dbo.Orders O ON C.CustomerID = O.CustomerID
GROUP BY C.CustomerID, C.CustomerName
ORDER BY TotalSpent DESC;



(3 rows affected)

Total execution time: 00:00:00.004

CustomerID CustomerName TotalSpent TotalOrders 5 Park & Shop Convenience Stores 17435,00 12 2 ABC Groceries 14870,00 9 3 ACME Bites 14706,00 10

## **Respondendo as Perguntas de Negócios**

### **Pergunta 2: Quais são os melhores e piores produtos da empresa?**

In [24]:
-- Identificando os produtos mais vendidos pelo soma das quantidades (Quantity)
SELECT TOP 6 
    P.CookieID, 
    P.CookieName, 
    SUM(OP.Quantity) AS TotalSold
FROM KCC.dbo.Product P
INNER JOIN KCC.dbo.Order_Product OP ON P.CookieID = OP.CookieID
GROUP BY P.CookieID, P.CookieName
ORDER BY TotalSold DESC;


(6 rows affected)

Total execution time: 00:00:00.001

CookieID CookieName TotalSold 2 Fortune Cookie 3468 5 Sugar 3110 1 Chocolate Chip 2937 4 Snickerdoodle 2751 3 Oatmeal Raisin 2494 6 White Chocolate Macadamia Nut 2489

In [26]:
-- Identificando os produtos menos vendidos
SELECT TOP 6 
    P.CookieID, 
    P.CookieName, 
    SUM(OP.Quantity) AS TotalSold
FROM KCC.dbo.Product P
INNER JOIN KCC.dbo.Order_Product OP ON P.CookieID = OP.CookieID
GROUP BY P.CookieID, P.CookieName
ORDER BY TotalSold ASC;


(6 rows affected)

Total execution time: 00:00:00.005

CookieID CookieName TotalSold 6 White Chocolate Macadamia Nut 2489 3 Oatmeal Raisin 2494 4 Snickerdoodle 2751 1 Chocolate Chip 2937 5 Sugar 3110 2 Fortune Cookie 3468

## **Respondendo as Perguntas de Negócios**

### **Pergunta 3: Existe sazonalidade nas vendas?**

In [28]:
-- Analisando a distribuição de pedidos por mês e valor total arrecadado
SELECT 
    MONTH(OrderDate) AS Month, 
    COUNT(OrderID) AS TotalOrders, 
    SUM(OrderTotal) AS TotalRevenue
FROM KCC.dbo.Orders
GROUP BY MONTH(OrderDate)
ORDER BY Month;


(3 rows affected)

Total execution time: 00:00:00.002

Month TotalOrders TotalRevenue 1 19 25021,00 2 22 29972,00 3 9 10898,00

## **Respondendo as Perguntas de Negócios**

### **Pergunta 4: Quais produtos devem ter aumento de preço e qual seria o novo preço?**

In [29]:
-- Recomendando produtos com base no custo e na receita por unidades vendidas
SELECT 
    P.CookieID, 
    P.CookieName, 
    SUM(OP.Quantity) AS TotalSold, 
    P.RevenuePerCookie, 
    P.CostPerCookie,
    (P.RevenuePerCookie - P.CostPerCookie) AS ProfitPerUnit,
    SUM(OP.Quantity) * (P.RevenuePerCookie - P.CostPerCookie) AS TotalProfit
FROM KCC.dbo.Product P
INNER JOIN KCC.dbo.Order_Product OP ON P.CookieID = OP.CookieID
GROUP BY P.CookieID, P.CookieName, P.RevenuePerCookie, P.CostPerCookie
ORDER BY TotalProfit DESC;


(6 rows affected)

Total execution time: 00:00:00.009

CookieID CookieName TotalSold RevenuePerCookie CostPerCookie ProfitPerUnit TotalProfit 1 Chocolate Chip 2937 5,00 2,00 3,00 8811,00 6 White Chocolate Macadamia Nut 2489 6,00 2,75 3,25 8089,25 3 Oatmeal Raisin 2494 5,00 2,20 2,80 6983,20 4 Snickerdoodle 2751 4,00 1,50 2,50 6877,50 5 Sugar 3110 3,00 1,25 1,75 5442,50 2 Fortune Cookie 3468 1,00 0,50 0,50 1734,00

## **Respondendo às Perguntas de Negócios e sugestões de melhorias**

 ### **A consulta SQL retornou os três melhores clientes, ordenados pelo total gasto (TotalSpent):**

1. **Park & Shop Convenience:** 17435,00 (12 pedidos)

2. **ABC Groceries:** 14870,00 (9 pedidos)

3. **ACME Bites:** 14706,00 (10 pedid

### **Sugestões de Ações Personalizadas:**


**1. Park & Shop Convenience:**

Campanhas: Criar uma campanha de marketing focada no excelente custo-benefício dos biscoitos, com destaque para promoções e ofertas especiais que realcem a relação entre qualidade e preço.

Comunicação Direta com Foco no Preço: Utilizar e-mails e outras formas de comunicação que enfatizem o valor acessível dos produtos, com promoções e pacotes especiais.


**2. ABC Groceries:**

E-mail Marketing Direto: Enviar e-mails concisos e informativos, com ofertas especiais e novidades, evitando mensagens muito longas.

Respostas Rápidas e Objetivas: Treinar a equipe para responder rapidamente aos contatos deste cliente com informações precisas e concisas, respeitando seu tempo.


**3. ACME Bites:**

Presente Personalizado: Enviar um pequeno presente com temática de rosas, demonstrando um cuidado especial e um entendimento profundo das preferências do cliente.

Comunicação Luxuosa e Refinada: Utilizar e-mails com design elegante e linguagem sofisticada, refletindo o bom gosto do cliente.





### **Com os resultados das consultas SQL, podemos responder à segunda pergunta de negócio e propor novas ideias de produtos.**

### **Resultados:**

**Melhores Produtos:** Os três melhores produtos, com base na soma das quantidades vendidas (TotalSold), são:

1. Fortune Cookie: 3468

2. Sugar Cookie: 3118

3. Chocolate Chip: 2937

**Piores Produtos:** Os três piores produtos são:

1. White Chocolate Macadamia: 2489

2. Oatmeal Raisin: 2494

3. Snickerdoodle: 2751

### **Insights:**

**Preferências do Consumidor:** Os resultados indicam uma clara preferência por sabores tradicionais e populares, como Fortune Cookie e Sugar Cookie. Chocolate Chip também se destaca, mostrando a força de um clássico.

**Desempenho Fraco:** Os três piores produtos têm vendas significativamente menores que os melhores, indicando a necessidade de mudanças.

**Oportunidade de Inovação:** A baixa performance dos piores produtos representa uma excelente oportunidade para a KCC investir em novos sabores que se alinhem às preferências do público e substituam os produtos menos vendidos.

### **Recomendações de Novos Sabores:**

Para substituir os piores produtos, recomendo focar em inovação e tendências, sem perder de vista a popularidade dos melhores:

**Substituindo White Chocolate Macadamia:** Este sabor, apesar de sofisticado, pode não ter atingido o público-alvo. Considerando a preferência por sabores mais tradicionais, sugiro experimentar um novo sabor com toque de chocolate mais intenso e popular, como um chocolate amargo com castanhas, ou um biscoito com gotas de chocolate meio amargo e pedaços de brownie.

**Substituindo Oatmeal Raisin:** O sabor de aveia com passas é muito específico. Para atingir um público maior, sugiro novos sabores com aveia, mas com outras combinações mais atraentes. Uma sugestão é um biscoito de aveia com pedaços de maçã e canela, ou um biscoito de aveia com chocolate branco e frutas secas.

**Substituindo Snickerdoodle:** Este sabor também é específico, e menos popular que outros. Uma possibilidade seria um sabor semelhante, mas com um toque diferente, como um biscoito amanteigado com pedaços de nozes, ou um biscoito com aroma de baunilha e raspas de limão.

**Considerações Adicionais:**

**Pesquisa de Mercado:** É fundamental realizar pesquisas de mercado para validar as sugestões de novos sabores e entender as preferências do público-alvo.

**Testes:** Antes do lançamento oficial, é crucial realizar testes com os novos sabores para coletar feedbacks e fazer ajustes antes da produção em larga escala.

**Marketing:** Uma campanha de marketing eficaz é fundamental para o sucesso dos novos produtos.Ótimo! Com os resultados das consultas SQL, podemos responder à segunda pergunta de negócio e propor novas ideias de produtos.


### **A consulta SQL, que mostra a distribuição de pedidos e receita por mês, ajuda na identificação da sazonalidade e propor campanhas de marketing.**

### **Insights:**

**Mês 2 (Fevereiro):** Apresenta o maior número de pedidos (22) e a maior receita (29972,00).

**Mês 1 (Janeiro):** Apresenta um número considerável de pedidos (19) e uma receita alta (25021,00).

**Mês 3 (Março):** Apresenta o menor número de pedidos (9) e a menor receita (10898,00).

### **Sazonalidade:**

Os dados sugerem uma clara sazonalidade nas vendas, com um pico em fevereiro e uma queda significativa em março. Para confirmar essa tendência, seriam necessários dados de mais meses, idealmente um ano inteiro. No entanto, baseado nos 3 meses apresentados, podemos assumir que há uma forte influência sazonal nas vendas.

### **Campanhas de Marketing:**

Considerando a sazonalidade observada, sugiro as seguintes campanhas de marketing:

**Campanha de Impulso em Março: Em março**, as vendas caíram drasticamente. Uma campanha de marketing agressiva nesse mês, focando em promoções e ofertas especiais, pode impulsionar as vendas e evitar uma queda tão significativa. Considerar ofertas promocionais, como descontos, brindes ou produtos em combo.

**Campanha de Fidelização em Janeiro e Fevereiro:** Considerando que Janeiro e Fevereiro são meses de alta venda, aproveitar a oportunidade para reforçar o relacionamento com os clientes e incentivar compras contínuas. Programas de fidelidade, ofertas personalizadas e conteúdos relevantes para fortalecer a marca podem ser interessantes.



### **Vamos analisar os resultados da consulta SQL para determinar quais produtos devem ter o preço aumentado e qual o novo preço ideal.**

### **Análise:**

A decisão de aumentar o preço de um produto deve considerar diversos fatores:

**Lucratividade:** Produtos com alta lucratividade por unidade (ProfitPerUnit) são candidatos ideais para aumento de preço, pois a margem de lucro permite absorver uma possível redução nas vendas.

**Demanda:** Produtos com alta demanda (TotalSold) podem suportar um pequeno aumento de preço sem afetar muito as vendas. No entanto, é crucial analisar a elasticidade-preço da demanda para cada produto.


### **Recomendações:**

Considerando os resultados, sugiro aumentar o preço dos seguintes produtos:

**Chocolate Chip:** Este produto apresenta um bom equilíbrio entre alta demanda (TotalSold: 2937) e alta lucratividade (ProfitPerUnit: 3,00). Um aumento de preço moderado, de 50 centavos, seria uma boa opção. 
- **Novo preço sugerido:** R$5,50. (Este aumento geraria um aumento significativo no lucro total, e não afetaria muito a demanda considerando seu bom desempenho atual.)

**White Chocolate Macadamia:** Apesar de ter vendas menores, este produto tem alta lucratividade (ProfitPerUnit: 3,25). Um aumento de preço pequeno, de R$ 0.50, é razoável para este produto.

- **Novo Preço Sugerido:** R$ 6..50   (Aumento moderado no lucro)

**Oatmeal Raisin:** Este produto apresenta uma lucratividade intermediária (ProfitPerUnit: 2,80), e vendas relativamente baixas. Um aumento de preço pode ser arriscado para este produto, pois o retorno pode ser mínimo. Por isso, deve-se avaliar cuidadosamente a receptividade do público a um aumento de preço, e o impacto que o aumento poderia causar na demanda. 

- **Novo preço sugerido:** R$ 5.50  (Um aumento pequeno, de 50 centavos, pode ser testado. Um aumento marginal no lucro, devendo ser monitorado de perto.)